vpr/src/place/move_generator.h

```diff
@@ -23,14 +23,19 @@ struct MoveOutcomeStats {
 /**
  * @brief A Struct to hold statistics about the different move types
```

Comment:
Holds statistics about the different move types performed so far in the placement.